db/kv.go

```diff
@@ -98,6 +99,21 @@ type secret struct {
 	LatestVersion api.SecretVersion
 }
 
+type byteString string
```

Comment:
Document why we need this type? Is there a trap with letting things serialize as normal strings?

Reply:
I will add a comment (I meant to anyway, and forgot). There is no risk, it's just that random binary crud will serialize as Unicode escapes, which is ugly.

Reply:
(done)